packages/alacritty/alacritty.pacscript

```diff
@@ -24,7 +24,7 @@ install() {
         sudo install -D extra/completions/_alacritty -t /usr/src/pacstall/${name}/usr/share/zsh/vendor-completions
 
         sudo mkdir -p /usr/src/pacstall/${name}/usr/local/share/man/man1
-        gzip -c extra/alacritty.man | sudo tee /usr/src/pacstall/${name}/usr/local/share/man/man1/alacritty.1.gz >/dev/null
+        sudo gzip -c extra/alacritty.man | sudo tee /usr/src/pacstall/${name}/usr/local/share/man/man1/alacritty.1.gz >/dev/null
```

Comment:
Why do you need sudo privileges to read the file `extra/alacritty.man`?
It's good security practice to limit the use of sudo to a absolute minimum.

Reply:
Yeah, I just tested it, the `sudo` is not needed, thanks for informing!

Reply:
Fixed it for both `alacritty` and `bat`